src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import commands.Command;
+import exceptions.DukeException;
+import parser.Parser;
+import storage.Storage;
+import tasklist.TaskList;
+import ui.Ui;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constuctor for Duke
+     */
+    public Duke() {
+        this.storage = new Storage();
+        this.ui = new Ui();
+        this.storage = new Storage();
+        ui.preload();
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            if (e.getMessage().equals("LoadFileError")) {
+                ui.loadFileError();
+            }
+        }
```

Comment:
Your try-catch statements follows the java coding standards. You also did this for several other places too. Well done!